includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```diff
@@ -718,8 +718,30 @@ public function maybe_process_upe_redirect() {
 				if ( isset( $_GET['redirect_status'] ) && 'succeeded' === $_GET['redirect_status'] ) {
 					$customer = new WC_Stripe_Customer( wp_get_current_user()->ID );
 					$customer->clear_cache();
-
 					wc_add_notice( __( 'Payment method successfully added.', 'woocommerce-gateway-stripe' ) );
+
+					// The newly created payment method does not inherit the customers' billing info, so we manually
+					// trigger an update; in case of failure we log the error and continue because the payment method's
+					// billing info will be updated when the customer makes a purchase anyway.
+					try {
+						$setup_intent_id = isset( $_GET['setup_intent'] ) ? wc_clean( wp_unslash( $_GET['setup_intent'] ) ) : '';
+						$setup_intent = WC_Stripe_API::request( [], 'setup_intents/' . $setup_intent_id, 'GET' );
```

Comment:
```suggestion
						$setup_intent_id = isset( $_GET['setup_intent'] ) ? wc_clean( wp_unslash( $_GET['setup_intent'] ) ) : '';
						$setup_intent    = WC_Stripe_API::request( [], 'setup_intents/' . $setup_intent_id, 'GET' );
```
My linter doesn't like the spacing and alignment here. I just want to make it clear here that my linter is making a complaint here, not me.